src/main/java/com/plugish/woominecraft/Util/BukkitRunner.java

```diff
@@ -1,25 +1,21 @@
 package com.plugish.woominecraft.Util;
 
 import com.plugish.woominecraft.WooMinecraft;
-import org.bukkit.ChatColor;
 import org.bukkit.scheduler.BukkitRunnable;
-import org.json.JSONException;
 
 public class BukkitRunner extends BukkitRunnable {
 
-	public final WooMinecraft plugin;
-
-	public BukkitRunner( WooMinecraft plugin ) {
-		this.plugin = plugin;
-	}
-
 	public void run() {
 		try {
-			plugin.check();
+			getPlugin().check();
 		} catch ( Exception e ) {
-			plugin.getLogger().warning( e.getMessage() );
+			getPlugin().getLogger().warning( e.getMessage() );
 			e.printStackTrace();
 		}
 	}
 
+	private WooMinecraft getPlugin() {

```

Comment:
@TekkitCommando Again, I don't see the benefit of creating a method in the sub-classes vs passing in the instance on the construct? Is it not better to just pass data around in Java vs creating a method? Method's do create memory overhead yea?